frontend/ui/src/pages/Channels/ChannelsMainPage/Sources/ChatPluginConnect.tsx

```diff
@@ -0,0 +1,296 @@
+    }
+  }, [props.channels, channelId]);
+
+  const createNewConnection = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    props
+      .connectChatPlugin({
+        name: displayName,
+        ...(imageUrl.length > 0 && {
+          imageUrl: imageUrl,
+        }),
+      })
+      .then((channel: Channel) => {
+        props.history.replace(CHANNELS_CHAT_PLUGIN_ROUTE + '/' + channel.id);
+      });
+  };
+
+  const updateConnection = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    props.updateChannel({channelId: channelId, name: displayName, imageUrl: imageUrl}).then(() => {
+      props.history.replace(CHANNELS_CHAT_PLUGIN_ROUTE);
+    });
+  };
+
+  const renderFormFields = () => (
+    <>
+      <div className={styles.formRow}>
+        <Input
+          type="text"
+          name="displayName"
+          value={displayName}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+            setDisplayName(e.target.value);
+          }}
+          label="Display Name"
+          placeholder="Add a name"
+          autoFocus
+          required
+          height={32}
+          fontClass="font-s"
+        />
+      </div>
+
+      <div className={styles.formRow}>
+        <Input
+          type="url"
+          name="url"
+          value={imageUrl}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+            setImageUrl(e.target.value);
+          }}
+          label="Image URL"
+          placeholder="(optionaly) add an image url"
+          hint="max. 1024x1024 pixel PNG"
+          height={32}
+          fontClass="font-s"
+        />
+      </div>
+    </>
+  );
+
+  const renderNewPage = () => {
+    return showNewPage ? (
+      <div>
+        <p className={styles.newPageParagraph}>Add Airy Live Chat to your website and application</p>
+
+        <Button type="button" onClick={() => setShowNewPage(false)}>
+          Connect Airy Live Chat
+        </Button>
+      </div>
+    ) : (
+      <div>
+        <p className={styles.newPageParagraph}>Add Airy Live Chat to your website and application</p>
+        <div className={styles.formWrapper}>
+          <div className={styles.settings}>
+            <form className={styles.form} onSubmit={createNewConnection}>
+              {renderFormFields()}
+              <Button type="submit" styleVariant="small">
+                Save
+              </Button>
+            </form>
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  const showSettings = (event: MouseEvent<HTMLAnchorElement>) => {
+    event.preventDefault();
+    setCurrentPage('settings');
+  };
+
+  const showInstallDoc = (event: MouseEvent<HTMLAnchorElement>) => {
+    event.preventDefault();
+    setCurrentPage('installDoc');
+  };
+
+  const generateCode = () => {
+    return `<script>
+  (function(w, d, s, n) {
+    w[n] = w[n] || {};
+    w[n].channelId = "${channelId}";
+    w[n].host = "${'https://' + env.API_HOST}";
```

Comment:
Ah sorry. Maybe I should have been more clear 😄 Right now the `API_HOST` is not the one hosting the chatplugin. So I meant to say that we'd have to inject another variable (`CHATPLUGIN_HOST`) on the `deployment.yaml`, the `nginx.conf` and finally the `env.ts`.

If you want me to do that, let me know. 

Also, we should leave out the protocol and only prepend `//` so that the script picks the same protocol as the host page.